lib/sqlitex/query.ex

```diff
@@ -0,0 +1,88 @@
+defmodule Sqlitex.Query do
+  use Pipe
+
+  defstruct bindings: [],
+            column_names: [],
+            column_types: [],
+            database: nil,
+            into: [],
+            raw_data: [],
+            sql: nil,
+            statement: nil
+
+  def query(db, sql, opts \\ []) do
+    pipe_matching {:ok, %Sqlitex.Query{}},
```

Comment:
I reran the Travis tests, and the same errors keep popping up:

```
       (sqlitex) lib/sqlitex/query.ex:63: Sqlitex.Query.prepare({:ok, %{__struct__: Sqlitex.Query}})

       (sqlitex) lib/sqlitex/query.ex:16: Sqlitex.Query.query/3
```

I feel like the `pipe_matching` macro is not interpreting the structure correctly.  A possible work-around until it can be investigated is to just match on `{:ok, _}`.